modules/apigee_edge_teams/src/EventSubscriber/TeamInvitationQueryAccessSubscriber.php

```diff
@@ -66,7 +66,14 @@ public function onQueryAccess(QueryAccessEvent $event) {
     // We query team from storage instead of check for a null team field because
     // the team might have been deleted on the remote server.
     $team_ids = array_keys($this->entityTypeManager->getStorage('team')->loadMultiple());
-    $event->getConditions()->addCondition('team', $team_ids);
+    if ($team_ids) {
+      $event->getConditions()->addCondition('team', $team_ids);
+    }
+    else {
+      // When no teams could be loaded from the storage, no invitation should be
+      // returned at all.
+      $event->getConditions()->alwaysFalse();
```

Comment:
Looks good to me, I do not see a direct test coverage for this class in the code, which concerns me. Maybe it is time to add one?

Reply:
Added a ticket to add test cases: #665